src/tracker/inflights.rs

```diff
@@ -121,6 +112,29 @@ impl Inflights {
     pub fn reset(&mut self) {
         self.count = 0;
         self.start = 0;
+        self.buffer = vec![];
+    }
+
+    /// Number of inflight messages.
+    #[inline]
+    pub fn count(&self) -> usize {
```

Comment:
How about making it doc hiddern as it's only used in tests?

Reply:
I got an error: missing documentation for an associated function